packages/grafana-runtime/src/config.ts

```diff
@@ -44,6 +44,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   ldapEnabled = false;
   sigV4AuthEnabled = false;
   samlEnabled = false;
+  samlName = 'SAML';
```

Comment:
Maybe `'SAML'` should be a constant that can be reused?

Reply:
I tried creating the constant in `app/core/constants`, but I don't think I can import it here. I also noticed that this value is never used, so maybe it would be best to make `samlName = ''` this way there would be less confusion in the future if we had to change this default value for some reason.

Reply:
I think we have a constants file here https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/constants.ts

Reply:
Nice! That works! I implemented this suggestion, but kept the `||` let me know what do you think 👍